mysql-test/suite/galera/t/galera_alter_table_hang.test

```diff
@@ -0,0 +1,55 @@
+#
+# Test ALTER TABLE hang due to CommitOrder and MDL hang
+#
+--source include/have_debug.inc
+--source include/galera_cluster.inc
+
+# Don't make threads wait
+SET GLOBAL wsrep_sync_wait=0;
+
+# Create auxiliary connections
+--connect(node_1_1, 127.0.0.1, root, , test, $NODE_MYPORT_1)
+--connect(node_1_2, 127.0.0.1, root, , test, $NODE_MYPORT_1)
```

Comment:
The convention we use is rather node_1a, node_1b or node_1_<specific_purpose> (like node_1_toi)

Reply:
Test case removed.